package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "screenshot-promise",
-  "version": "1.1.0",
+  "version": "1.1.1",
```

Comment:
We all use [np](https://github.com/sindresorhus/np) to publish our packages, which actually does quite a lot more then only bumping the version. If you are only interested in how to bump the version, you could do that with [npm version](https://docs.npmjs.com/cli/version) (e.g. `npm version patch`).

Also, it's just best practice to have a commit that only bumps the current version and that you can tag. This way you will always have clean commit history that is self explanatory. [Here](https://github.com/sindresorhus/np/commits/master) is an example.